### PR TITLE
Dropped safeguards against very old versions of gettext.

### PR DESCRIPTION
**Pull request by claudep**
gettext 0.19 was released in 2014.

### CODE REVIEW

**Comment by claudep** [comment]
Mmmh, I found some more possible deletions. Updates follow...

File: docs/topics/i18n/translation.txt [hunk 1]
```diff
@@ -1551,7 +1551,7 @@ Django comes with a tool, :djadmin:`django-admin makemessages
     commands from the GNU gettext toolset: ``xgettext``, ``msgfmt``,
     ``msgmerge`` and ``msguniq``.
 
-    The minimum version of the ``gettext`` utilities supported is 0.15.
+    The minimum version of the ``gettext`` utilities supported is 0.19.
```
Comment (felixxm) [inline docs/topics/i18n/translation.txt:1554]: I'd add a release note in the `Backwards incompatible changes in 5.2` section.

Comment (claudep) [reply docs/topics/i18n/translation.txt:1554]: I hesitated to do it, but did not as the removed version support is so old. I guess it doesn't hurt, so I can add it anyway.

Comment (claudep) [reply docs/topics/i18n/translation.txt:1554]: Done.

**Review by felixxm — APPROVED** [review]
@claudep Thanks :+1: Looks great :rocket:

**Comment by claudep** [comment]
Thanks for the review!

**Pull request merged**